ramls/instance-bulk.raml

```diff
@@ -0,0 +1,50 @@
+#%RAML 1.0
+title: Inventory Storage Instance Bulk API
+version: v0.1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost
+
+documentation:
+  - title: Inventory Storage Instance Bulk Download API
+    content: <b>API for downloading a bulk set of Instance IDs</b>
+
+types:
+  id: !include id.json
+  ids: !include ids.json
+  errors: !include raml-util/schemas/errors.schema
+
+traits:
+  language: !include raml-util/traits/language.raml
+  orderable: !include raml-util/traits/orderable.raml
+  searchable: !include raml-util/traits/searchable.raml
+  validate: !include raml-util/traits/validation.raml
+
+resourceTypes:
+  collection-get: !include raml-util/rtypes/collection-get.raml
+
+/instance-bulk:
+  /ids:
+    displayName: Instance ID Bulk Download API
+    type:
+      collection-get:
+        schemaCollection: ids
+        exampleCollection: !include examples/ids.json
+    get:
+      description: Retrieve a list of Instance IDs.
+      is: [
+        searchable: {description: "with valid searchable fields", example: "name=aaa"},
+        validate
+      ]
+      queryParameters:
+        field:
+          description: Field
+          type: string
+          enum: [id]
+          default: id
+          required: false
```

Comment:
I'd remove this property at all for now, it can be added once required. What do you think?

Reply:
The plan is to extend it for `field=hrid`. This is also an id therefore `/instance-bulk/ids` is a reasonable API path name.

Reply:
The MODINVSTOR-439 states that HRID is out of scope now, so I don't see any benefit in adding a flag that is not used. Also, requirements is not clear enough, maybe it will be ok to have both ID + HRID in the response.

Reply:
I was thinking _Can take a query parameter to allow HRIDs in the future_ meant that I had to provide the initial infrastructure like this field query parameter, even if it only support `id` at this time.
As for _ID + HRID_, I just want to confirm, are you suggesting retrieving multiple columns per row (id column, hrid column)?

Reply:
Ok, let's leave it for now, since it is optional.